src/main/java/org/jenkinsci/plugins/workflow/steps/EnvironmentExpander.java

```diff
@@ -45,6 +46,29 @@
     public abstract void expand(@Nonnull EnvVars env) throws IOException, InterruptedException;
 
     /**
+     * Provides an expander for a constant map of string keys and string values.
+     *
```

Comment:
Should link to `EnvVars.override` so it is clear this is handling `PATH+XYZ` notation.